pages/linux/ssh-add.md

```diff
@@ -0,0 +1,24 @@
+# ssh-add
+
+> Manage loaded ssh keys in the ssh-agent.
+> Ensure that ssh-agent is up and running for the keys to be loaded in it.
+
+- Add the standard ssh key, e.g. {{id_rsa}}, to the ssh-agent:
```

Comment:
It actually adds ~/.ssh/id_rsa, ~/.ssh/id_dsa, ~/.ssh/id_ecdsa, ~/.ssh/id_ed25519 and ~/.ssh/identity. But that would bloat the description. 

Maybe "Add the default ssh keys from the "~/.ssh" directory to the ssh-agent" sounds better.